burn.py

```diff
@@ -0,0 +1,27 @@
+"""A class for storing the burn pile on the poker table, cards in it are stored in a list and not intended to be shown
+during the course of the game"""
+
+from card import Card
+from deck import Deck
+
+class Burn:
+    def __init__(self, burn_pile):
+        self.burn_pile = []
```

Comment:
Burn __init__ function does not need burn_pile as a positional argument since it initializes to empty (i.e., the value gets ignored) so please take it out.  Then Burn can be referenced by Burn() and not Burn([]).